styles/globals.css

```diff
@@ -121,3 +121,17 @@ main li {
 main li span {
   flex: 0 0 7rem; 
 }
+
+.favorite-pin {
```

Comment:
It would be better to position the pin absolute to the heading
<img width="434" alt="image" src="https://user-images.githubusercontent.com/16056918/199091599-c6255472-5bae-447c-b162-6d70371f1d8b.png">

Also, same as with the remove button, better to hide the pins and only show with 0.5 opacity only when hovering the heading